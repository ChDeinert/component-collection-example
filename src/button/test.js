import React from 'react';
import renderer from 'react-test-renderer';
import Button from './';

describe('Button', () => {
  it('renders correctly', () => {
    const renderedTree = renderer.create(<Button>Content</Button>);
    expect(renderedTree.toJSON()).toMatchSnapshot();
  });
});
