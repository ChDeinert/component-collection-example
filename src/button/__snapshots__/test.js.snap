// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  onClick={undefined}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "border": "1px solid #eee",
      "borderRadius": 3,
      "cursor": "pointer",
      "fontSize": 15,
      "margin": 10,
      "padding": "3px 10px",
    }
  }
>
  Content
</button>
`;
